admin/app/components/solidus_admin/orders/cart/component.html.erb

```diff
@@ -0,0 +1,115 @@
+<div
+  class="w-full relative overflow-visible"
+  data-controller="<%= stimulus_id %>"
+  data-<%= stimulus_id %>-products-url-value="<%= solidus_admin.variants_for_order_path(@order) %>"
+  data-action="
+    keydown-><%= stimulus_id %>#selectResult
+  "
+>
+  <%= render component('ui/panel').new do |panel| %>
+    <div>
+      <div class="peer">
+      <%= render component("ui/forms/search_field").new(
```

Comment:
Are there any plans to make this reusable components? We currently have the `productPicker` and `variantSelect` that can be used outside of the admin cart. Would be great if we could keep this functionality in the future.

Reply:
> The general plan also relies on the expectation that the building blocks we'll end up with at the end of the projects will be good enough and generic enough to be useful for custom interfaces as well. This approach doesn't exclude doing a "final" pass on the components looking for additional extractions or missing pieces but we want to give priority to sourcing them from concrete needs rather than imagining how we'll use them in the future.

Very well thought. Thanks for explaining!